source/system-administrators/activities/debugging-ssl-issues.rst

```diff
@@ -0,0 +1,66 @@
+:is-up-to-date: True
+
+.. _debugging_ssl_issues:
+
+==============================
+Debugging SSL Handshake Issues
+==============================
+
+In this section, we present an error that a user may encounter regarding SSL and how to fix the error.
+
+Java has decided to `disable support for TLS 1.0 and 1.1 <https://bugs.java.com/bugdatabase/view_bug.do?bug_id=JDK-8202343>`__ in the latest Java version 1.8.0_292 as indicated in the
```

Comment:
Please change to:
"Starting with JVM version 1.8.0_292, support for TLS v1.1 and earlier is disabled by default."